starknet_py/tests/e2e/fixtures/accounts.py

```diff
@@ -0,0 +1,223 @@
+# pylint: disable=redefined-outer-name
```

Comment:
Assuming this was just moved from `account_clients`?

Reply:
Probably yes.

Reply:
The file was just renamed